test/move-count.test.js

```diff
@@ -0,0 +1,33 @@
+import axios from './__mocks__/axios';
+import count from '../src/modules/movies-count';
+
+describe('Count total movies', () => {
+  let movieCountElement;
+
+  beforeEach(() => {
+    // Create a dummy movieCount element in the document body
+    movieCountElement = document.createElement('div');
+    movieCountElement.className = 'movie-count';
+    document.body.appendChild(movieCountElement);
+  });
+
+  afterEach(() => {
+    // Remove the movieCount element from the document body after each test
+    movieCountElement.remove();
+  });
+
+  it('test_api_call_failure', async () => {
+    const mockAxios = jest.spyOn(axios, 'get').mockRejectedValue(new Error('API call failed'));
+    const consoleSpy = jest.spyOn(console, 'error').mockImplementation();
+    await count();
+    expect(mockAxios).toHaveBeenCalledWith('https://api.tvmaze.com/shows/1/episodes');
+    expect(consoleSpy).toHaveBeenCalledWith('Error:', new Error('API call failed'));
+  });
+
+  it('test_api_response_data_not_in_expected_format', async () => {
```

Comment:
- You have done well so far, but I have an error when I try running your tests, Please fix this 🙏 
![t5t34t5](https://github.com/Izebiafe/BleFi/assets/73607512/45ad2a1a-38e5-4792-bf4b-0992097d0b40)

- Also after implementing a proper counter for the movie count, please add the correct tests 😄